profilegen/src/res/templates/spec_template.template

```diff
@@ -0,0 +1,30 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+
+require_relative 'lib/client'
```

Comment:
I am not sure whether we need this or not?

Reply:
We need this